kbs/src/lib.rs

```diff
@@ -11,6 +11,7 @@ extern crate anyhow;
 extern crate base64;
 extern crate env_logger;
 extern crate kbs_types;
+#[allow(unused_imports)]
```

Comment:
why this?

Reply:
the compiler triggers an unused_imports warning when buiding without `as`. See the commit message https://github.com/confidential-containers/trustee/pull/458/commits/83fa39013576a86e98faadef3e5b0e5b4af81d28

Reply:
I suggest not to use macro mod-level, but the concrete place it warns. Or, this would let future unused-imports error not be raised.

Reply:
I can drop it. It has nothing to do with this PR.

Reply:
@Xynnn007 btw, this is exactly where the place is:

```
warning: unused `#[macro_use]` import
  --> kbs/src/lib.rs:14:1
   |
14 | #[macro_use]
   | ^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

warning: `kbs` (lib) generated 1 warning
```

Reply:
I dropped this and submitted and issue #480